### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/calc#262]
## Results for https://calc.gsa.gov:
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > header > div > nav > ul > li > a
  - `<a href="/about/">About this tool</a>`
- **Error:** Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
  - WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
  - html > body > header > div > div > div > div > div:nth-child(1) > img
  - `<img class="ceiling-icon" src="/static/hourglass_site/images/price-callout/roof.png">`
- **Error:** Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
  - WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
  - html > body > header > div > div > div > div > div:nth-child(2) > img
  - `<img class="fully-burdened-icon" src="/static/hourglass_site/images/price-callout/mule.png">`
- **Error:** Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
  - WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
  - html > body > header > div > div > div > div > div:nth-child(3) > img
  - `<img src="/static/hourglass_site/images/price-callout/breifcase.png">`
- **Error:** Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
  - WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
  - html > body > header > div > div > div > div > div:nth-child(4) > img
  - `<img src="/static/hourglass_site/images/price-callout/globe.png">`
- **Error:** Form does not contain a submit button (input type="submit", input type="image", or button type="submit").
  - WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
  - #search
  - `<form id="search" class="search" role="form"> <input type="hidden" name="s...</form>`
- **Error:** This fieldset element does not have a name available to an accessibility API. Valid names are: legend element.
  - WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Fieldset.Name
  - #search > section:nth-child(4) > div > div > div:nth-child(1) > div > dl > dd > div > fieldset > fieldset
  - `<fieldset> </fieldset>`
- **Error:** Fieldset does not contain a legend element. All fieldsets should contain a legend element that describes a description of the field group.
  - WCAG2AA.Principle1.Guideline1_3.1_3_1.H71.NoLegend
  - #search > section:nth-child(4) > div > div > div:nth-child(1) > div > dl > dd > div > fieldset > fieldset
  - `<fieldset> </fieldset>`
- **Error:** Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
  - WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
  - #standard-deviation-minus-highlight
  - `<h5 id="standard-deviation-minus-highlight" class="sd-highlight"></h5>`
- **Error:** Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
  - WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
  - #avg-price-highlight
  - `<h5 id="avg-price-highlight"></h5>`
- **Error:** Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
  - WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
  - #standard-deviation-plus-highlight
  - `<h5 id="standard-deviation-plus-highlight" class="sd-highlight"></h5>`
- **Error:** Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
  - WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
  - #search > section:nth-child(4) > div > div > div:nth-child(2) > div:nth-child(7) > div:nth-child(2) > h5:nth-child(2)
  - `<h5 class="proposed-price-highlight"></h5>`
## Summary:
- 12 Errors

**Comment by nickbristow** [comment]
These results can be recreated using the HTML_Codesniffer applet

**Comment by toolness** [comment]
During CI, we're running aXe-core on the data explorer as of #1552 and the styleguide as of #1554.  Given that it's comparable to pa11y and this issue is pretty old--we've fixed tons of accessibility bugs since then--I'm going to close this issue.
